docs/reference/contributing/target/QuadSPI.md

```diff
@@ -1,25 +1,50 @@
 <h3 id="quadspi-port">QuadSPI</h3>
```

Comment:
Sometimes, we use "QuadSPI," and sometimes we use "QSPI". Which is the API in the code?

Reply:
QSPI is an object in the drivers API The full name should be Quad Serial Peripheral Interface. 

This then should be changed to QSPI?